dist/origin-web-catalogs.css

```diff
@@ -34,9 +34,6 @@
     opacity: 0;
   }
 }
-.landing-main-area {
-  padding: 20px;
```

Comment:
Wonder if we shouldn't give a temporary margin to the headings if we're removing the padding on the entire page.